oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -1316,6 +1316,7 @@ private void collectDeletedProperties(final NodeDocument doc, final GCPhases pha
                         .sum();
 
                 deletedPropsCountMap.put(doc.getId(), deletedPropsCount);
+                fullGCStats.collectedPropertiesDeleted(GCPhase.FULL_GC_COLLECT_PROPS, deletedPropsCount);
```

Comment:
this stats is wrong because we might end up in a situation where the property collected for deletion during the `COLLECT` phase is not at all deleted.

Reply:
I have changed the names of the statistics to clarify it is counting the candidates to delete. Those numbers can then mismatch the actually deleted statistics, since they could be skipped in later stages.